sync_diff_inspector/config.go

```diff
@@ -162,5 +185,68 @@ func (c *Config) checkConfig() bool {
 		return false
 	}
 
+	if len(c.SourceDBCfg) == 0 {
+		log.Error("must have at least one source database")
+		return false
+	}
+
+	sourceDBMap := make(map[string]DBConfig)
+	for i := range c.SourceDBCfg {
+		if c.SourceDBCfg[i].Label == "" {
+			// add label for source database
+			c.SourceDBCfg[i].Label = fmt.Sprintf("%s:%d", c.SourceDBCfg[i].Host, c.SourceDBCfg[i].Port)
+		}
+		sourceDBMap[c.SourceDBCfg[i].Label] = c.SourceDBCfg[i]
+	}
+
+	if len(c.SourceDBCfg) > 1 {
+		if len(c.Tables) == 0 {
+			log.Error("must specify check tables if have more than one source")
+			return false
+		}
+	}
+
+	for _, table := range c.Tables {
+		if table.Schema == "" {
+			table.Schema = c.TargetDBCfg.Schema
+		}
+
+		// setting Range to "TRUE" can make the code more simple, no need to judge the Range's value.
+		// for example: sql will looks like "select * from itest where a > 10 AND TRUE" if don't set range in config.
+		if table.Range == "" {
+			table.Range = "TRUE"
+		}
+
+		if len(table.SourceTables) == 0 {
+			if len(c.SourceDBCfg) > 1 {
```

Comment:
in fact,  u use `if len(c.SourceDBCfg) > 1` as `isSharding`, can we explicitly use an isSharding parameter and then specify the configuration format， we can easy to write user document and review it.